attributes/default.rb

```diff
@@ -89,7 +89,7 @@
 default['kube-hops']['kfserving']['enabled']               = node['install']['kubernetes']
 default['kube-hops']['kfserving']['version']               = "0.5.1"
 default['kube-hops']['kfserving']['base_dir']              = node['kube-hops']['dir'] + "/kfserving"
-default['kube-hops']['kfserving']['img_tar_url']           = node['download_url'] + "/kube/kfserving/kfserving-v#{node['kube-hops']['kfserving']['version']}.tgz"
+default['kube-hops']['kfserving']['img_tar_url']           = node['download_url'] + "/kube/kfserving/#{node['hopsworks']['version']}/kfserving-v#{node['kube-hops']['kfserving']['version']}.tgz"
```

Comment:
Don't use node['hopsworks']['version'] - it is not a Berksfile dependency.
Use this instead: node['install']['version']

Reply:
Done